src/Type/Php/StrlenFunctionReturnTypeExtension.php

```diff
@@ -30,8 +33,16 @@ public function getTypeFromFunctionCall(
 		}
 
 		$argType = $scope->getType($args[0]->value);
+
+		if ($argType instanceof ConstantStringType) {
```

Comment:
You could use `TypeUtils::getConstantStrings()` and return correct values even for unions of constant strings.